homeassistant/components/unifi/sensor.py

```diff
@@ -55,6 +64,22 @@ def add_entities(controller, async_add_entities, clients):
         async_add_entities(sensors)
 
 
+@callback
+def add_uptime_entities(controller, async_add_entities, clients):
```

Comment:
Maybe try and reuse `add_entities` callback by adding a parameter for the type of entity? We can create a dict that maps the entity type to the entity class.

Reply:
I think it is fine like this, it mirrors all other platforms in UniFi integration, makes it easier to navigate